guacamole-docker/bin/start.sh

```diff
@@ -403,6 +403,142 @@ END
 
 }
 
+# Print error message regarding missing required variables for SQLServer authentication
+sqlserver_missing_vars() {
+    cat <<END
+FATAL: Missing required environment variables
+-------------------------------------------------------------------------------
+If using a SQLServer database, you must provide each of the following
+environment variables or their corresponding Docker secrets by appending _FILE
+to the environment variable, and setting the value to the path of the
+corresponding secret:
+
+    SQLSERVER_USER     The user to authenticate as when connecting to
+                       SQLServer.
+
+    SQLSERVER_PASSWORD The password to use when authenticating with SQLServer
+                       as SQLSERVER_USER.
+
+    SQLSERVER_DATABASE The name of the SQLServer database to use for Guacamole
+                       authentication.
```

Comment:
Similar to the documentation section, above, this should also probably be reworked to take into account using either these three variables or the `_FILE` versions for Docker secret support. Again, I know it matches the Postgres and MySQL versions, so those should be reworked later, but since we're adding these, here, I think it makes sense to do it correctly.

Reply:
Updated too, hopefully it's ok.